doc/release/1.3.0-notes.rst

```diff
@@ -37,6 +37,15 @@ Deprecated features
 Backwards incompatible changes
 ==============================
 
+Functions from ``scipy.interpolate`` (``spleval``, ``spline``, ``splmake``,
+and ``spltopp``) and functions from ``scipy.misc`` (``bytescale``, ``fromimage``,
+``imfilter``, ``imread``, ``imresize``, ``imrotate``, ``imsave``, ``imshow``,
+``toimage``) have been removed. The former set has been deprecated since v0.19.0
+and the latter has been deprecated since v1.0.0.
+Similarly, aliases from ``scipy.misc`` which have been deprecated since v1.0.0 are
```

Comment:
could you spell out the name of these aliases?

Reply:
Added the names.